File: src/main/java/com/alan/service/CacheService.java
package com.alan.service;

/**
 * 缓存操作
 * @Author: 孟庆浩
 * @Description:
 * @Date: Created in 18-1-4 下午10:03
 * @Modified By:
 */
public interface CacheService {

    public <V> V cacheResult(String key, String cacheName);

    public void cacheRemove(String key, String cacheName);

    public <V> void cachePut(String key, V value, String cacheName);
}

File: src/main/java/com/alan/service/CacheService.kt
package com.alan.service

/**
 * 缓存操作
 * @Author: 孟庆浩
 * @Description:
 * @Date: Created in 18-1-4 下午10:03
 * @Modified By:
 */
interface CacheService {

    fun <V> cacheResult(key: String, cacheName: String): V

    fun cacheRemove(key: String, cacheName: String)

    fun <V> cachePut(key: String, value: V, cacheName: String)
}
